src/mocks.ts

```diff
@@ -152,6 +152,12 @@ export class InventoryDataMock {
       resolve();
     });
   }
+
+  public return(): any {
+    return new Promise((resolve, reject) => {
```

Comment:
I think you can replace this with `return Promise.resolve()`, but I'm not sure. This change, though minor, would reduce the line count of your mocks file by a third. 😄